packages/cx-navigation/src/components/MenuTitle/MenuTitleClean.tsx

```diff
@@ -1,10 +1,12 @@
 import { EditorPlainClean } from '@bodiless/cx-editors';
 import { asCxTokenSpec } from '@bodiless/cx-elements';
+import { LinkClean } from '@bodiless/cx-link';
 import { replaceable, replaceWith, withDesign } from '@bodiless/fclasses';
 import type { MenuTitleComponents } from '@bodiless/navigation';
 import { MenuTitle } from '@bodiless/navigation';
 
 const MenuTitleClean = withDesign({
+  Link: replaceWith(LinkClean),
```

Comment:
Are we 100% sure we want this in the clean menu title?  will this be used in the main menu and burger menu, or just in the footer menu?

Reply:
@wodenx ,

The links don't have the external/download icons. By adding the LinkClean to the MenuTitle link component, it only changes its structure to the external/download tokens can be properly applied.